src/ContentHubClient.php

```diff
@@ -955,127 +955,106 @@ public function __call($method, $args)
-                case 'createEntity':
-                    return $this->getErrorResponse(500,
-                      sprintf('Error trying to create an entity in Content Hub (Error Message: %s)', $exception->getMessage()));
-
-                case 'createEntities':
-                    return $this->getErrorResponse(500,
-                      sprintf('Error trying to create entities in Content Hub (Error Message = %s)', $exception->getMessage()));
-
-                case 'updateEntity':
-                    return $this->getErrorResponse(500,
-                      sprintf('Error trying to update entity with UUID="%s" in Content Hub (Error Message = %s)', $args[1],
-                        $exception->getMessage()));
-
-                case 'updateEntities':
-                    return $this->getErrorResponse(500,
-                      sprintf('Error trying to update some entities in Content Hub (Error Message = %s)',
-                        $exception->getMessage()));
-
-                case 'deleteEntity':
-                    return $this->getErrorResponse(500,
-                      sprintf('Error trying to delete entity with UUID="%s" in Content Hub (Error Message = %s)', $args[0],
-                        $exception->getMessage()));
-
-                case 'searchEntity':
-                    return $this->getErrorResponse(500,
-                      sprintf('Error trying to make a search query to Content Hub. Are your credentials inserted correctly? (Error Message = %s)',
-                        $exception->getMessage()));
-
-                default:
-                    return $this->getErrorResponse(500,
-                      sprintf('Error trying to connect to the Content Hub. Are your credentials inserted correctly? (Error Message = %s)',
-                        $exception->getMessage()));
-            }
+
+        switch ($method) {
+            case 'getClientByName':
+                // All good, means the client name is available.
+                if ($response && 404 === $status_code) {
+                    return $response;
+                }
+
+                $message = sprintf('Error trying to connect to the Content Hub" (Error Code = %d: %s)',
+                    $status_code, $error_details);
+                break;
+
+            case 'addWebhook':
+                $message = sprintf('There was a problem trying to register Webhook URL = %s. Please try again. (Error Code = %d: %s)',
+                    $args[0], $status_code, $error_details);
+                break;
+
+            case 'deleteWebhook':
+                // This function only requires one argument (webhook_uuid), but
+                // we are using the second one to pass the webhook_url.
+                $webhook_url = isset($args[1]) ? $args[1] : $args[0];
+                $message = sprintf('There was a problem trying to <b>unregister</b> Webhook URL = %s. Please try again. (Error Code = %d: @%s)',
```

Comment:
As far as I know, it's a long time that using `b` to make the text bold is deprecated (same goes true for `i` and alike). Not sure if we can avoid using it here, too.